### DISCUSSION TITLE
Text highlight not working on windows version

### DISCUSSION THREAD

**Discussion by skossan** [jesseduffield/lazygit#1465] [Q&A] [answered]
HI!
I'm using lazygit on my windows machine I don't know which branch I am on, same geos for the list of commits. There is no highlight so I don't what which item is in focus

**Comment by mjarkk** [comment, marked as answer]
Have you read [/docs/Config.md#struggling-to-see-selected-line](https://github.com/jesseduffield/lazygit/blob/master/docs/Config.md#struggling-to-see-selected-line)  
To edit the config file you can go to the status pannel and press `e`

**Reply by skossan** [reply]
That worked perfectly! Thanks!
